### DISCUSSION TITLE
安装hexo并使用了 git一键部署功能（hexo-deployer-git），但是部署的时候，总是自动部署至master分支，是什么原因呢？

### DISCUSSION THREAD

**Discussion by Voyiid** [hexojs/hexo#5550] [Question / Answer] [answered]
remote: Resolving deltas: 100% (7/7), completed with 4 local objects.
To https://github.com/Voyiid/voyiid.github.io
   1bd1121..8a9da21  HEAD -> master
分支 'main' 设置为跟踪 'https://github.com/Voyiid/voyiid.github.io/master'。
INFO  Deploy done: git

**Comment by uiolee** [comment, marked as answer]
https://github.com/hexojs/hexo-deployer-git#readme

You can configure it according to the readme
